notes/10-fsea.jl

```diff
@@ -137,3 +137,84 @@ subset!(fsdf, :pvalue=> ByRow(!isnan))
 fsdf.qvalue = adjust(fsdf.pvalue, BenjaminiHochberg())
 sort!(fsdf, :qvalue)
 CSV.write(datafiles("fsea_o12.csv"))
+
+
+#-
+
+brain_features = Symbol.(replace.(Resonance.brainmeta, "-"=>"_"))
```

Comment:
I don't think this solves the whole problem. You're supposed to be able to run W-longform.jl with the sole output of the first notebooks, so maybe notebook 10 is not the exact best place to do it. Plus, there's a conflict of sorts: I haven't looked into it in detail, but it seems that method `startup`, as I understand, needs the hyphenated version, but it does a substitution operation on lines 127-128:

```
    rename!(tps, Dict(k=> replace(k, "-"=>"_") for k in brainmeta))
    brainmeta = map(i-> replace(brainmeta[i], "-"=>"_"), eachindex(brainmeta))
```

So maybe the correct way of fixing it is to take a closer look inside the startup.jl instead of making changes to high-numbered noteboooks?

Reply:
just remember that `brainmeta` is supposed to be a `const` so we have to pay attention to to the way we "fix" it.

Reply:
A lot of this is a remnant (in my brain, not so much in the code) of before DataFrames allowed you to do things like df."Left-Thalamus", and everything had to be converted to `Symobl`. I should probably start wrapping my head around strings - it's been available for over a year.

Note though that the `const` variable is mostly to reference into the raw dataset, and the `brainmeta = ...` there is reassigning it in function scope, not actually modifying the constant. And I don't think this should block merging for now - worth opening an issue though.

Reply:
Ok. I'll mark as resolved and approve thee changes then.